### ISSUE TITLE
[Accessibility]: CodeMirror blob view search input loses focus when typing with screenreader enabled

### ISSUE THREAD

**Issue by taras-yemets** [sourcegraph/sourcegraph#45621]
### Audit type

Screen reader navigation

### Problem description

When typing in the CodeMirror blob view search field it often loses focus. To continue typing user needs to focus the input field again.

https://user-images.githubusercontent.com/25318659/207336370-ee313a7b-625c-4c21-9278-c77e1cd6df52.mov

### Expected behavior

Typing in the search field is not interrupted.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

/cc @felixfbecker
